src/Libraries/RevitServices/Threading/RevitSchedulerThread.cs

```diff
@@ -42,6 +42,9 @@ private void OnRevitIdle(object sender, IdlingEventArgs e)
                 return;
             }
 
+            if (revitApplication.ActiveUIDocument == null)
```

Comment:
It seems this would mean that without a revit host doc open, user will not be able to actually run the graph. You may want to update the warning message and add comments here.